src/app/types/SubmitBar.ts

```diff
@@ -12,10 +12,18 @@ export interface StateProps {
   maps: SubmissionInterfaces.Map[];
   debugRunAvailable: boolean;
   aiIds: number[];
+  isStoryModeModalOpen: boolean;
+  storyModeModalLevel: number;
+  // tslint:disable-next-line
+  ratings: { level: number; stars: number }[];
```

Comment:
Use put this in a custom type and use a custom type array?
Why use typescript if you always choose to ignore the rules?
Always, always listen to the typescript compiler.